cloud: add SQL and TiFlash related docs (#9001) (#9479)
pingcap/docs: table-attributes.md

When the above two attributes are configured at the same time, the Regions belonging to the partition `p` can actually be merged. When the attribute of the partition is reset, the partition `p` inherits the attribute from the table `t`, and the Regions cannot be merged.

<CustomContent platform="tidb">

> **Note:**
>

> > - When there is now only an attribute of a partition, even if the `merge_option=allow` attribute is configured, the partition is still split into multiple Regions by default according to the actual number of partitions. To merge all Regions, you need to [reset the attribute of the table](#usage).

Reviewer: ```suggestion
> - When only the partition attribute is used, even if you configure `merge_option=allow`, the partition is still split into multiple Regions by default according to the actual number of partitions. To merge all Regions, you need to [reset the attribute of the table](#usage).
```

Reviewer 2: According to the feature owner @rleungx of this doc, "When there is now only an attribute of a partition" means "When only the attribute of the table `t` is used", so the original translation is misleading.

> - When using the `merge_option` attribute, you need to pay attention to the PD configuration parameter [`split-merge-interval`](/pd-configuration-file.md#split-merge-interval). Suppose that the `merge_option` attribute is not configured. In this case, if Regions meet conditions, Regions can be merged after the interval specified by `split-merge-interval`. If the `merge_option` attribute is configured, PD decides whether to merge Regions after the specified interval according to the `merge_option` configuration.

</CustomContent>

<CustomContent platform="tidb-cloud">

> **Note:**
>

> > - When there is now only an attribute of a partition, even if the `merge_option=allow` attribute is configured, the partition is still split into multiple Regions by default according to the actual number of partitions. To merge all Regions, you need to [reset the attribute of the table](#usage).

Reviewer: ```suggestion
> - When only the partition attribute is used, even if you configure `merge_option=allow`, the partition is still split into multiple Regions by default according to the actual number of partitions. To merge all Regions, you need to [reset the attribute of the table](#usage).
```
